src/lcd.c

```diff
@@ -137,6 +139,32 @@ void myGPIOB_Init(void){
     GPIO_Init(GPIOB, &GPIO_InitStruct);
 }
 
+void LCD_CustomHzSymbol(void){
+  char customChar[8] = {
+    0b10001,
+    0b11111,
+    0b10001,
+    0b01110,
+    0b00010,
+    0b00100,
+    0b01000,
+    0b01110
+  };
+  // resets the RS and R/W pins
+  LCD_SendWord(LCD_COMMAND, 0x00);
```

Comment:
I was able to comment out this line and L165 and it still works. AFAIK L156 does all the work of setting the CGRAM location and flagging the next 8 words as lines for the custom symbol.